index.js

```diff
@@ -226,6 +293,78 @@ Massive.prototype.documentTableSql = function(tableName){
   return sql;
 };
 
+
+Massive.prototype.dropDocumentTable = function(table, options, next) {
```

Comment:
Oh, no, `createDocumentTable` should definitely stay as-is -- it's just `dropDocumentTable` will work equally well on a document or any other table, so there's no need to call it out specifically.